### DISCUSSION TITLE
Gitter down, matrix web access?

### DISCUSSION THREAD

**Discussion by uvtc** [janet-lang/janet#901] [General]
Seems like gitter is currently down. Is there another way to use/view the chat via some sort of Matrix web access? Or maybe just via good ol' IRC?

**Comment by pepe** [comment]
https://matrix.to/#/#janet-language_community:gitter.im is the Matrix room.

**Comment by uvtc** [comment]
Thanks, Pepe. I'd like to understand better how Janet uses Matrix...
